maestrowf/interfaces/script/_flux/flux0_26_0.py

```diff
@@ -37,6 +35,9 @@ def submit(
         # single node, don't use a broker -- but introduce a flag that can
         # force a single node to run in a broker.
 
+        if isinstance(ngpus, str) and ngpus.isdigit():
```

Comment:
Is this type/isdigit checking still required after the changes to the upper-level fluxscriptadaptor?

Reply:
Good catch -- missed this one.